source/appModules/explorer.py

```diff
@@ -177,6 +179,14 @@ def _get_windowText(self):
 			return windowText.replace(CHAR_LTR_MARK,'').replace(CHAR_RTL_MARK,'')
 		return windowText
 
+
+class metadataEditField(RichEdit50):

```

Comment:
Note that class names start in upper case

```suggestion
class MetadataEditField(RichEdit50):
```

Reply:
Mistakes like this are  perfect candidates for automatic detection by linter. Would it be possible to implement this somehow?